components/network/samba/patches/11-avahi-register.patch

```diff
@@ -0,0 +1,20 @@
+--- samba-4.9.5/source3/smbd/avahi_register.c	2019-03-12 09:19:04.000000000 +0000
```

Comment:
Please, add reference to https://www.illumos.org/issues/10577 on the top of this patch.

Reply:
Will do that.